source/set-mutable.d.ts

```diff
@@ -0,0 +1,35 @@
+import {Except} from './except';
+import {Mutable} from './mutable';
+
+/**
+Create a type that converts the given keys from `readonly` to mutable. The remaining keys are kept as is.
+
+Use-case: You want to define a single model where the only thing that changes is whether or not some of the keys are mutable.
+
```

Comment:
Can you add a `@see` tag for the `Mutable` type. And the inverse in the `Mutable` type file?